docs/logs/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,98 @@
+# Getting Started with OpenTelemetry .NET Logs in 5 Minutes - ASP.NET Core Application
+
+First, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new web application:
+
+```sh
+dotnet new web -o aspnetcoreapp
+cd aspnetcoreapp
+```
+
+Install the
+[OpenTelemetry.Exporter.Console](../../../src/OpenTelemetry.Exporter.Console/README.md)
+and
+[OpenTelemetry.Extensions.Hosting](../../../src/OpenTelemetry.Extensions.Hosting/README.md)
+packages:
+
+```sh
+dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Extensions.Hosting
+```
+
+Copy the [HelloLogs.cs](./HelloLogs.cs) and [Program.cs](./Program.cs) files to
+the project folder.
+
+Run the application again (using `dotnet run`) and then browse to the URL shown
+in the console for your application (e.g. `http://localhost:5154`). You should see
+the logs output from the console.
+
+```text
+LogRecord.Timestamp:               2023-09-01T23:36:24.3541759Z
+LogRecord.CategoryName:            Microsoft.Hosting.Lifetime
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.Body:                    Content root path: {contentRoot}
+LogRecord.Attributes (Key:Value):
+    contentRoot: D:\repo\opentelemetry-dotnet\docs\logs\getting-started-aspnetcore\
+    OriginalFormat (a.k.a Body): Content root path: {contentRoot}
+
+Resource associated with LogRecord:
+service.name: getting-started-aspnetcore
+service.instance.id: d4f09b09-6613-4d09-8a2d-1508e4ce9067
+telemetry.sdk.name: opentelemetry
+telemetry.sdk.language: dotnet
+telemetry.sdk.version: 1.6.0-rc.1.16
+
+LogRecord.Timestamp:               2023-09-01T23:36:52.6956536Z
+LogRecord.TraceId:                 abb98d33a390d2773985fd3488333e14
+LogRecord.SpanId:                  46e671d30d10df22
+LogRecord.TraceFlags:              None
+LogRecord.CategoryName:            Program
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.Body:                    Food `{name}` price changed to `{price}`.
+LogRecord.Attributes (Key:Value):
+    name: artichoke
+    price: 9.99
+    OriginalFormat (a.k.a Body): Food `{name}` price changed to `{price}`.
+LogRecord.EventId:                 1
+LogRecord.EventName:               FoodPriceChanged
+LogRecord.ScopeValues (Key:Value):
+[Scope.0]:SpanId: 46e671d30d10df22
```

Comment:
neutral about this part... given this is beginner/getting-started guide, maybe don't enable scopes at all here, to keep it really simple.

Reply:
especially since each exporter handles scope differently. (OTLP does not prefix "Scope.depth" to scopes, but console does....)

Reply:
I would leave the scopes enabled in the sample code under the assumption that many developers will want them and discovering them when they are missing from the sample is much harder. I do think it would be fine to abbreviate the output more.

Reply:
We have traditionally kept getting started as the minimal thing to get started. (https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started-aspnetcore/Program.cs#L27-L29 - it does not even customize sampler which most people do in their real apps).
Scopes is a common use case, but not necessarily a requirement for getting-started.

Reply:
Decided to keep the scopes.